nva-datamodel-java/src/main/java/no/unit/nva/model/instancetypes/artistic/film/realization/MovingPictureOutput.java

```diff
@@ -6,9 +6,9 @@
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
 @JsonSubTypes({
-        @JsonSubTypes.Type(name = "Broadcast", value = Broadcast.class),
-        @JsonSubTypes.Type(name = "CinematicRelease", value = CinematicRelease.class),
-        @JsonSubTypes.Type(name = "OtherRelease", value = OtherRelease.class)
+    @JsonSubTypes.Type(name = "Broadcast", value = Broadcast.class),
```

Comment:
General comment: Better if the type name is contained the class and then you can have 
```
@JsonSubTypes.Type(name = Broadcast.TYPE, value = Broadclas.class)
```